app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -114,13 +114,13 @@ class GithubWebhookSubscription extends SubscriptionHandler {
         }
         break;
       case 'push':
-        final PushEvent pushEvent = PushEvent.fromJson(json.decode(webhook.payload) as Map<String, dynamic>);
-        final String branch = pushEvent.ref!.split('/')[2]; // Eg: refs/heads/beta would return beta.
-        final String repository = pushEvent.repository!.name;
+        final Map<String, dynamic> event = jsonDecode(webhook.payload) as Map<String, dynamic>;
+        final String branch = event["ref"].split('/')[2]; // Eg: refs/heads/beta would return beta.
```

Comment:
Typically we use single quotes. Only use double quotes if your string contains single quotes.

Reply:
https://dart.dev/tools/linter-rules/prefer_single_quotes

Reply:
Thanks! I updated the instances of double quotes in my PR.